tests/drivers/flash_nop_device/prj.conf

```diff
@@ -6,4 +6,3 @@
 CONFIG_ZTEST=y
 CONFIG_FLASH=y
 CONFIG_FLASH_SIMULATOR=n
```

Comment:
Do we need this test ?

Reply:
We should probably hardwire the module source file, the flash_nop_device.c, to the test.

Reply:
This is only disabling the flash simulator.
I mentioned earlier that this was not needed, but @de-nordic prefered to keep it for safety.
https://github.com/nrfconnect/sdk-nrf/pull/5620#discussion_r709158638

Reply:
so it has no direct relation to the now removed `FLASH_NOP_DEVICE` setting,

Reply:
Disabling flash simulator is fine and it's actually needed. Problem is the fact that `drivers/flash/flash_nop_device.c` was moved and it would not be compiled as part of the test. So as Dominik mentioned, we at least need the hardwired source code to let test pass. In the current state the test does not include source file of the tested driver.

Reply:
> Problem is the fact that drivers/flash/flash_nop_device.c was moved and it would not be compiled as part of the test. 

Right, now I get it :wink: 
We would need the test itself to add this file.